activesupport/lib/active_support/message_verifier.rb

```diff
@@ -23,7 +23,7 @@ module ActiveSupport
   #     self.current_user = User.find(id)
   #   end
   #
-  # By default it uses Marshal to serialize the message. If you want to use
+  # By default it uses JSON to serialize the message. If you want to use
```

Comment:
There should likely be some mention of the config flag here instead of just saying it uses JSON

Reply:
No longer valid with the change to how the flag works